apps/example-app/app/screens/LoggingScreen.tsx

```diff
@@ -0,0 +1,171 @@
+
+export const LoggingScreen: React.FC<LoggingScreenProps> = function LoggingScreen() {
+  const ref = React.useRef<ViewShot>(null)
+
+  const handleScreenshot = () => {
+    captureRef(ref, {
+      format: "jpg",
+      quality: 0.8,
+      result: "data-uri",
+    }).then(
+      (uri) => {
+        if (__DEV__) {
+          console.tron.display({
+            name: "Screenshot",
+            preview: "App screenshot",
+            image: { uri },
+          })
+        }
+      },
+      (error) => console.error("Oops, snapshot failed", error),
+    )
+  }
+
+  return (
+    <Screen style={$container} preset="scroll">
+      <ViewShot ref={ref}>
+        <View style={$topContainer}>
+          <Text style={$text} tx="loggingsScreen.title" />
+        </View>
+        <View style={{ marginTop: spacing.lg }}>
+          <Button
+            text="console.log"
+            textStyle={$darkText}
+            style={$button}
+            onPress={() => {
+              console.log("Hello from Reactotron!")
+            }}
+          />
+          <Button
+            text="console.log with data"
+            textStyle={$darkText}
+            style={$button}
+            onPress={() => {
+              console.log("Hello from Reactotron!", { this: "is", some: "data" })
+            }}
+          />
+          <Button
+            text="console.debug"
+            textStyle={$darkText}
+            style={$button}
+            onPress={() => {
+              console.debug("This is a console.debug()")
+            }}
+          />
+          <Button
+            text="console.warn"
+            textStyle={$darkText}
+            style={$button}
+            onPress={() => {
+              console.warn("This is a console.warn()")
+            }}
+          />
+          <Button
+            text="console.error"
+            textStyle={$darkText}
+            style={$button}
+            onPress={() => {
+              console.error("This is a console.error()", { with: "possible data" })
+            }}
+          />
+        </View>
+        <View style={$topContainer}>
+          <Text style={$text} tx="loggingsScreen.subtitle" />
+        </View>
+        <View style={{ marginTop: spacing.lg }}>
+          <Button
+            text="console.tron.image"
+            textStyle={$darkText}
+            style={$button}
+            onPress={() => {
+              if (__DEV__) {
+                console.tron.image({
+                  uri: "https://placekitten.com/g/400/400",
+                  preview: "placekitten.com",
+                  filename: "cat.jpg",
+                  width: 400,
+                  height: 400,
+                  caption: "D'awwwwwww",
+                })
+              }
+            }}
+          />
+          <Button
+            text="Log a Screenshot"
+            textStyle={$darkText}
+            style={$button}
+            onPress={handleScreenshot}
+          />
```

Comment:
Appears that this does not respect dark mode as my screenshot has a white background instead of a black one.

Reply:
I think it's because the background color is on the top container and not the View within the `<ViewShot>`. I'll fix.